pkg/kubelet/cm/dra/claiminfo.go

```diff
@@ -179,11 +186,18 @@ func (cache *claimInfoCache) add(res *ClaimInfo) {
 	cache.claimInfo[res.ClaimName+res.Namespace] = res
 }
 
-func (cache *claimInfoCache) get(claimName, namespace string) *ClaimInfo {
+// getClaimInfo returns a deep copy of the cached claimInfo object
+// for the specified claim name and namespace
+func (cache *claimInfoCache) getClaimInfo(claimName, namespace string) (ClaimInfo, error) {
```

Comment:
Why the name change? claimInfoCache.GetClaimInfo() seems unnecessarily redundant.

Reply:
ClaimInfoCache also contains state and usually used as m.cache, so it's not always obvious what exactly get function returns if you look at the manager code. I can change it back if you think it doesn't make sense or can rename to say getClaimInfoCopy to emphasise that it returns a clone/copy. WDYT?

Reply:
I don't buy the argument about containing a `state` variable. The `state` variable is internal and only used for syncing claimInfos to the checkpoint. The purpose of the abstraction is to cache claimInfo objects via get/put calls, so calling the function `get()` seems appropriate and correct.

If you think calling `m.cache.get()` is not clear, then I'd argue for a change in the `cache` variable name, not the function call name, i.e. `m.claimInfoCache.get()`